# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/FetchEventOpParent.h
# Commit: 8fec60d0c499
# Full Hash: 8fec60d0c499fdf429f4d11da6bea0c65aecb1d4
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2022-02-27 09:40:54
# Regressor Bug: 1754365
# File Overlap Count: 1
# Description:
#   Bug 1754365 - Create IPC for transferring NavigationPreload's PerformanceTimingData. r=dom-worker-reviewers,jesup
#   
#   Depends on D138250
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138251
# ==============================================================================

diff -r fb0ac558803d -r 8fec60d0c499 dom/serviceworkers/FetchEventOpParent.h
--- a/dom/serviceworkers/FetchEventOpParent.h	Sun Feb 27 00:26:21 2022 +0000
+++ b/dom/serviceworkers/FetchEventOpParent.h	Sun Feb 27 00:26:22 2022 +0000
@@ -25,7 +25,7 @@
 
   // Transition from the Pending state to the Started state. Returns the preload
   // response, if it has already arrived.
-  Maybe<ParentToParentInternalResponse> OnStart(
+  Maybe<ParentToParentResponseWithTiming> OnStart(
       MovingNotNull<RefPtr<FetchEventOpProxyParent>> aFetchEventOpProxyParent);
 
   // Transition from the Started state to the Finished state.
@@ -37,12 +37,12 @@
   // IPDL methods
 
   mozilla::ipc::IPCResult RecvPreloadResponse(
-      ParentToParentInternalResponse&& aResponse);
+      ParentToParentResponseWithTiming&& aResponse);
 
   void ActorDestroy(ActorDestroyReason) override;
 
   struct Pending {
-    Maybe<ParentToParentInternalResponse> mPreloadResponse;
+    Maybe<ParentToParentResponseWithTiming> mPreloadResponse;
   };
 
   struct Started {